openfe/setup/load_molecules.py

```diff
@@ -0,0 +1,46 @@
+import pathlib
+from openfe import SmallMoleculeComponent
+from rdkit import Chem
+
+def _smcs_from_sdf(sdf):
+    supp = Chem.SDMolSupplier(str(sdf), removeHs=False)
+    mols = [SmallMoleculeComponent(mol) for mol in supp]
+    return mols
+
+def _smcs_from_mol2(mol2):
+    rdmol = Chem.MolFromMol2File(str(mol2), removeHs=False)
+    return [SmallMoleculeComponent.from_rdkit(rdmol)]
+
+
+def load_molecules(file_or_directory):
```

Comment:
also maybe something like `load_SmallMoleculeComponents` would clarify what molecules this function handles